apps/web/app/(authenticated)/(app)/app/settings/root-keys/[keyId]/layout.tsx

```diff
@@ -35,7 +35,7 @@ export default async function Layout({ children, params: { keyId } }: Props) {
   return (
     <div className="flex flex-col gap-4">
       <Card>
-        <CardContent className="grid grid-cols-5 divide-x">
+        <CardContent className="flex justify-between flex-wrap divide-x">
```

Comment:
why is this changing styling?

the pr is about a file rename, no?

Reply:
applogies for that, was solving the issue #885 sorry for the mistake :(.